VagrantProvision.sh

```diff
@@ -43,119 +64,183 @@ if ! grep --quiet "export JAVA_HOME" ~/.profile; then
     cd gdal-1.10.1
-    sudo ./configure --with-fgdb=/usr/local/FileGDB_API --with-pg=/usr/bin/pg_config --with-python
-    sudo make -j$(nproc)
-    sudo make install
+    echo "GDAL: configure"
+    sudo ./configure --quiet --with-fgdb=/usr/local/FileGDB_API --with-pg=/usr/bin/pg_config --with-python
+    echo "GDAL: make"
+    sudo make -sj$(nproc) > GDAL_Build.txt 2>&1
+    echo "GDAL: install"
+    sudo make -s install >> GDAL_Build.txt 2>&1
     cd swig/python
-    python setup.py build
-    sudo python setup.py install
+    echo "GDAL: python build"
+    python setup.py build >> GDAL_Build.txt 2>&1
+    echo "GDAL: python install"
+    sudo python setup.py install >> GDAL_Build.txt 2>&1
     sudo ldconfig
     cd ~
 fi
 
 if ! grep --quiet NODE_PATH ~/.profile; then
-    echo "Installing node js dependencies..."
+    echo "### Installing node js dependencies..."
     sudo npm config set registry http://registry.npmjs.org/
-    sudo npm install -g xml2js htmlparser imagemagick mocha@1.20.1 express@3.1.2 async html-to-text restler
+    sudo npm install --silent -g xml2js htmlparser imagemagick mocha@1.20.1 express@3.1.2 async html-to-text restler
     echo 'Adding NODE_PATH to user environment...'
     echo 'export NODE_PATH=/usr/local/lib/node_modules' >> ~/.profile
     source ~/.profile
 fi
 
 # Module needed for OSM API db test
 if [ ! -d $HOME/.cpan ]; then
-    echo "Installing XML::Simple module..."
+    echo "### Installing Perl XML::Simple module..."
     (echo y;echo o conf prerequisites_policy follow;echo o conf commit)|sudo cpan
-    sudo perl -MCPAN -e 'install XML::Simple'
+    sudo perl -MCPAN -e 'install XML::Simple' > /dev/null
+    sudo chown -R vagrant:vagrant $HOME/.cpan
 fi
 
+# NOTE: These have been changed to pg9.5
 if ! sudo -u postgres psql -lqt | grep -i --quiet hoot; then
-    echo "Creating Services Database..."
+    echo "### Creating Services Database..."
     sudo -u postgres createuser --superuser hoot
     sudo -u postgres psql -c "alter user hoot with password 'hoottest';"
     sudo -u postgres createdb hoot --owner=hoot
     sudo -u postgres createdb wfsstoredb --owner=hoot
     sudo -u postgres psql -d hoot -c 'create extension hstore;'
-    sudo -u postgres psql -d postgres -c "UPDATE pg_database SET datistemplate='true' WHERE datname='wfsstoredb'"
-    sudo -u postgres psql -d wfsstoredb -f /usr/share/postgresql/9.1/contrib/postgis-1.5/postgis.sql
-    sudo -u postgres psql -d wfsstoredb -f /usr/share/postgresql/9.1/contrib/postgis-1.5/spatial_ref_sys.sql
+    sudo -u postgres psql -d postgres -c "UPDATE pg_database SET datistemplate='true' WHERE datname='wfsstoredb'" > /dev/null
+    sudo -u postgres psql -d wfsstoredb -f /usr/share/postgresql/9.5/contrib/postgis-2.2/postgis.sql > /dev/null
+    sudo -u postgres psql -d wfsstoredb -f /usr/share/postgresql/9.5/contrib/postgis-2.2/spatial_ref_sys.sql > /dev/null
     sudo -u postgres psql -d wfsstoredb -c "GRANT ALL on geometry_columns TO PUBLIC;"
```

Comment:
We should use this syntax here
`sudo -u postgres psql -d wfsstoredb -c 'create extension postgis;'`
instead of calling sql files directly